src/graphql/schema/Climb.gql

```diff
@@ -154,6 +154,7 @@ type GradeType {
   """
   ewbank: String
   french: String
+  brazilian_crux: String
```

Comment:
I was not 100% sure if this should be `brazilian` or `brazilian_crux`.

Reply:
This is correct